.github/workflows/ci.yml

```diff
@@ -199,7 +196,6 @@ jobs:
           directory: empty
           token: 87cefb17-c44b-4f2f-8b30-1fff5769ce46
           name: Alpine
-          flags: Alpine,3.12
```

Comment:
By the way, computing the Python version dynamically is usually better instead of tracking hardcoded values..
When `setup-python` is used, it provides an output. Otherwise, `python -V` is your friend...